enterprise/frontend/src/embedding-sdk-bundle/store/reducer.ts

```diff
@@ -61,9 +61,7 @@ export const getOrRefreshSession = createAsyncThunk(
     const state = getSessionTokenState(getState() as SdkStoreState);
     const token = storedAuthToken ?? state?.token;
 
-    const isTokenValid = token && token.exp * 1000 >= Date.now();
-
-    if (isTokenValid) {
+    if (isSessionTokenValid(token)) {
```

Comment:
Ok, so here's really interesting. Before we don't expect JWT to not contain `exp`, now we do. if we cdon't fix this line. We will never return existing token.

```ts
    if (isTokenValid) {
      return token;
    }
```

So, we will just keep hitting this function over and over again. I tested locally that we will just keep calling the SSO endpoint.

This fix works by using the same underlying function we use when validating the token in the first place. This makes the logic more centralized and easy to reason about.